modularizar.py

```diff
@@ -0,0 +1,45 @@
+import datetime
+
+
+def obter_limite():
+    NOME_COMPLETO = "Rodiney Branta"
+    print(f"Seja bem-vindo a Poppy loja , sou {NOME_COMPLETO}, entre e escolha um produto\n")
+    ano_atual = datetime.datetime.today()
+    cargo_atual = str(input("Para fazermos sua análise de crédito, precisamos que você entre com algumas informações:\nCargo atual:"))
```

Comment:
Não misture as coisas. Como havia comentado antes, se deixar os textos misturados, para dar manutenção depois fica pior. Dar boas vindas é uma coisa, solicitar o cargo atual é outra. Veja o [single-responsibility principle](https://en.wikipedia.org/wiki/Single-responsibility_principle) do [S.O.L.I.D.](https://en.wikipedia.org/wiki/SOLID)
 
```suggestion
    print(f"Para fazermos sua análise de crédito, precisamos que você entre com algumas informações:")
    cargo_atual = str(input("Cargo atual:"))
```

Reply:
> Não misture as coisas. Como havia comentado antes, se deixar os textos misturados, para dar manutenção depois fica pior. Dar boas vindas é uma coisa, solicitar o cargo atual é outra. Veja o [single-responsibility principle](https://en.wikipedia.org/wiki/Single-responsibility_principle) do [S.O.L.I.D.](https://en.wikipedia.org/wiki/SOLID)

Aceito a sugestão, fiz a leitura do  single-responsibility principle e vou seguir as regras do S.O.L.I.D. em meus códigos.